Corona-Warn-App/proguard-rules.pro

```diff
@@ -123,3 +123,5 @@
 -keep class de.rki.coronawarnapp.covidcertificate.person.ui.admission.model.** { *; }
 -keep class de.rki.coronawarnapp.covidcertificate.revocation.model.** { *; }
 -keep class de.rki.coronawarnapp.datadonation.** { *; }
+-keep class de.rki.coronawarnapp.covidcertificate.valueset.valuesets.** { *; }
+-keep class de.rki.coronawarnapp.covidcertificate.vaccination.core.repository.storage.** { *; }
```

Comment:
Please build a `release` build and make sure nothing is broken by Pro-guard ?

Reply:
No -keep present from that branch after switching to Gson